docs/api/asyncio_con.rst

```diff
@@ -234,8 +234,51 @@ Connection
             If the results of *query* are desired, :py:meth:`query` or
             :py:meth:`query_one` should be used instead.
 
+    .. py:method:: retry()
+
+        Create a :py:class:`AsyncIORetry` object. This is a way to make
+        reliable transactions. See
+        :ref:`edgedb-python-asyncio-api-transaction` for more info.
+
+        Example:
+
+        .. code-block:: python
+
+            async for tx in con.retry():
+                with tx:
+                    value = tx.query_one("SELECT Counter.value")
+                    tx.execute(
+                        "UPDATE Counter SET { value := <int64>$value",
+                        value=value,
+                    )
+
+        Note executing queries on ``tx`` object rather than the original
+        connection.
+
+    .. py:method:: try_transaction()
+
+        Create an individual :py:class:`AsyncIOTransaction` object.
+
+        Transaction may fail so it's often better to use :py:meth:`retry`
+        instead. This method is targeted for users who want to implement \
+        more specialized transaction retry loop.
```

Comment:
```suggestion

        Execute a non-retryable transaction.
        
        Contrary to ``retry()``, ``try_transaction()`` will not attempt
        to re-run the nested code block in case a retryable error happens.
        
        This is a low-level API and it is advised to use the ``retry()`` 
        method instead.

        A call to ``try_transaction()`` returns 
        :py:class:`AsyncIOTransaction`.
```